end-session/t/01_end-session_run.pl

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env perl
+
+use v5.010;
+use strict;
+use warnings;
+
+use lib '../perl';
+
+use BookerTest qw(test_sql_statement clean_dir_sub run_tests fail booker_init
+	          booker_create_session);
+use Cwd;
+
+my $HOME = $ENV{HOME};
+my $cd = getcwd;
+
+run_tests(
+    { name  => "Normal session end",
+      pre   => [ \&booker_init, \&booker_create_session ],
+      cmd   => ["./booker-end-session"],
+      tests => [ test_sql_statement(
+                    "counting special session users",
+                    "$HOME/.booker",
+                    "SELECT COUNT(*) FROM session_user WHERE user = 'user'",
```

Comment:
I dont think a good test here is to count users, it is probably better to check the session has an end